### PR TITLE
add a state sync script and add syncing to readme.md

### PR DESCRIPTION
**Pull request by faddat**
This PR adds statesync script and adds state sync to the readme file so that users can easily join the neutron network.

### CODE REVIEW

**Review by dragonfrond — APPROVED** [review]

**Review by foxpy — COMMENTED** [review]
This looks cool actually

File: contrib/statesync.bash [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+set -uxe
+
+# Set Golang environment variables.
+# ! Adapt as required, depending on your system configuration
+#export GOPATH=~/go
+#export PATH=$PATH:~/go/bin
+
+# Install with pebbledb (uncomment for incredible performance)
+go mod edit -replace github.com/tendermint/tm-db=github.com/baabeetaa/tm-db@pebble
+go mod tidy
```
Comment (albertandrejev) [inline contrib/statesync.bash:19]: I am not sure that we need to modify sources in this script even in this way. It is good to use on local installation, but there is risk that this changes will fall into sources. And it makes neutrond client incompatible with available snapshots. This requires additional discussion and decision.

File: contrib/statesync.bash [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+# go install -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/types.DBBackend=pebbledb -X github.com/tendermint/tm-db.ForceSync=1' -tags pebbledb ./...
+
+# install (comment if using pebble for incredible performance)
+# go install ./...
+
+
+go install -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/types.DBBackend=pebbledb -X github.com/tendermint/tm-db.ForceSync=1' -tags pebbledb ./...
```
Comment (albertandrejev) [inline contrib/statesync.bash:27]: same here

File: contrib/statesync.bash [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+export NEUTROND_NODE=tcp://127.0.0.1:8011
+export NEUTROND_P2P_MAX_NUM_INBOUND_PEERS=500
+export NEUTROND_RPC_PPROF_LADDR=127.0.0.1:6969
+
+# Fetch and set list of seeds from chain registry.
+NEUTROND_P2P_SEEDS=$(curl -s https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/chain.json | jq -r '[foreach .peers.seeds[] as $item (""; "\($item.id)@\($item.address)")] | join(",")')
+export NEUTROND_P2P_SEEDS
+
+# Start chain.
+neutrond start --x-crisis-skip-assert-invariants --iavl-disable-fastnode false
```
Comment (albertandrejev) [inline contrib/statesync.bash:73]: I suspect that you forgot to put here something to modify `config.toml` using above variables.

File: contrib/statesync.bash (hunk 1 above)
Comment (faddat) [reply contrib/statesync.bash:19]: we don't, technically

File: contrib/statesync.bash (hunk 3 above)
Comment (faddat) [reply contrib/statesync.bash:73]: we don't need to modify config.toml

File: contrib/statesync.bash (hunk 2 above)
Comment (faddat) [reply contrib/statesync.bash:27]: make sense

**Pull request closed without merging**